src/content-script/ChatGPTFeedback.tsx

```diff
@@ -42,8 +45,34 @@ function ChatGPTFeedback(props: Props) {
     })
   }, [props, action])
 
+  const clickCopyToClipboard = useCallback(async () => {
+    setAction('copyToClipboard')
+    setCopyToClipboard(true)
+    if ('clipboard' in navigator) {
+      await navigator.clipboard.writeText(props.answerText)
+    } else {
+      return document.execCommand('copy', true, props.answerText)
```

Comment:
I think there is no need to do this, since all browsers this extension supports should have `navigator.clipboard` support

Reply:
Oh you're right
https://developer.mozilla.org/en-US/docs/Web/API/Navigator/clipboard#browser_compatibility

removed that